### DISCUSSION TITLE
Frepple Tools keep disappearing in odoo

### DISCUSSION THREAD

**Discussion by Michaelatastra** [frePPLe/frepple#636] [Q&A]
about once a day the frepple toolbar that usually appears just under the Odoo toolbar just vanishes.  Logging out and logging back in to Odoo works about half the time.  Sometimes like today I cant get the tools back at all.  If I am logged out for 5-8 hours it will return

**Comment by jdetaeye** [comment]
Not 100% what's going on, but here are some hints:
- When using the "advanced planning frepple" option in the odoo toolbar, the frepple toolbar should always show up at the top of the frepple screen.
- The other places where the frepple screens are integrated in the odoo toolbar should not display the frepple toolbar. They just show the frepple content without the toolbar.
- Of course, you can still log in frepple directly, without first logging in in odoo. The frepple toolbar always displays then.
- Things can get tricky if you have multiple browser tabs open with a mix of the methods above... Your browser session may then get confused whether or not the toolbar needs to be displayed or not.

**Comment by Michaelatastra** [comment]
Hi Johan,
Thanks for the suggestions
The issue occurs with the advanced planning tab. In fact this is usually where the issue encountered.
Exactly the same results if I go directly to frepple outside of Odoo
I will try working with a single window today to see if there is a difference in behavior




Thanks



[Image]

Michael Rothberg

Production Scheduling – Machine Shop

E: ***@***.*** | M: 415.272.2286

He/Him/His

PROPRIETARY DATA STATEMENT AND ITAR EXPORT CONTROL WARNING

The information contained in this email and any files/attachments transmitted with it are proprietary to Astra Space, Inc. and are intended only for the addressee.  Access to this email by anyone else is unauthorized.  If you receive this transmission in error, please contact the sender immediately and delete the material from your computer.

Additionally, this email and/or its attachments may contain technical data whose export is restricted by the U.S. Department of State under the International Traffic in Arms Regulations (ITAR) or Department of Commerce Export Administration Regulations (EAR).  Licenses approved by the Department of State and/or Commerce are required prior to the export of ITAR/EAR-controlled technical data to any foreign person or organization; either in the United States or abroad.

________________________________
From: Johan De Taeye ***@***.***>
Sent: Wednesday, February 21, 2024 11:50:23 PM
To: frePPLe/frepple ***@***.***>
Cc: Michael Rothberg ***@***.***>; Author ***@***.***>
Subject: [Ext] Re: [frePPLe/frepple] Frepple Tools keep disappearing in odoo (Discussion #636)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Not 100% what's going on, but here are some hints:

  *   When using the "advanced planning frepple" option in the odoo toolbar, the frepple toolbar should always show up at the top of the frepple screen.
  *   The other places where the frepple screens are integrated in the odoo toolbar should not display the frepple toolbar. They just show the frepple content without the toolbar.
  *   Of course, you can still log in frepple directly, without first logging in in odoo. The frepple toolbar always displays then.
  *   Things can get tricky if you have multiple browser tabs open with a mix of the methods above... Your browser session may then get confused whether or not the toolbar needs to be displayed or not.

—
Reply to this email directly, view it on GitHub<https://github.com/frePPLe/frepple/discussions/636#discussioncomment-8553211>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BGK26OV4IZTIMGM7KQC7JJLYU32D7AVCNFSM6AAAAABDUCYYIKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNJTGIYTC>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

CAUTION: This email originated from outside the organization. Do not trust the identity of the sender, the safety of any links, or any attachments unless you know the content is safe.
